### PR TITLE
Feature/field names

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**  
Update fields to the following:
image

image

**Why**  
To get the relevant information from the user

**Anything else?**

### CODE REVIEW

File: templates/createDropbox.hbs [hunk 1]
```diff
@@ -59,73 +59,40 @@
...
-        } else {
-          document.querySelector('#identifiable-fields-warning').classList.remove('visually-hidden')
-          return false;
-        }
-      }
-    </script>
-  </section>
+<section>
+  <h2>2. Add your details:</h2>
+  <p>
```
Comment (mmmmillar) [inline templates/createDropbox.hbs]: I don't think this paragraph is relevant anymore

**Review by mmmmillar — COMMENTED** [review]
Looks good, will wait to hear back from David Court on possible text changes

File: templates/createDropbox.hbs (hunk 1 above)
Comment (kosiakkatrina) [reply templates/createDropbox.hbs]: ah, good shout!

File: lib/use-cases/SaveDropbox.js [hunk 2]
```diff
@@ -22,26 +22,22 @@ module.exports = ({ gateways: { dropbox: dropboxes } }) => {
...
+      dob,
+      lastName,
+      parentsEmail,
       archived
     } = fields;
 
     dropbox.description = validValueOrNull(description);
 
-    if (isValidNonEmptyString(customerName)) {
+    if (isValidNonEmptyString(firstName)) {
```
Comment (mmmmillar) [inline lib/use-cases/SaveDropbox.js]: This validation of one field in the use-case isn't really great, I think it'd be enough to validate all our required fields on the client-side using patterns

see https://stackoverflow.com/questions/13766015/is-it-possible-to-configure-a-required-field-to-ignore-white-space?lq=1

File: lib/use-cases/SaveDropbox.js [hunk 3]
```diff
@@ -22,26 +22,22 @@ module.exports = ({ gateways: { dropbox: dropboxes } }) => {
...
-      customerDob,
-      customerEmail,
-      customerPhone,
-      customerReference,
-      customerNationalInsurance,
+      firstName,
+      dob,
+      lastName,
+      parentsEmail,
       archived
```
Comment (mmmmillar) [inline lib/use-cases/SaveDropbox.js]: Archived can be removed as is not used

File: templates/createDropbox.hbs [hunk 4]
```diff
@@ -58,48 +58,30 @@
   </p>
   <form action="" method="POST" onsubmit="return validateIdentification();">
     <div class="form-group">
-      <label for="customerName">Your name</label>
-      <input type="text" class="form-control" id="customerName" name="customerName" value="{{ dropbox.customer.name }}"
+      <label for="firstName">Child’s First Name</label>
```
Comment (mmmmillar) [inline templates/createDropbox.hbs]: Only capitalise the first word of these labels to match the rest of the app

File: templates/createDropbox.hbs [hunk 5]
```diff
@@ -59,73 +59,37 @@
...
-      }
-    </script>
-  </section>
+<section>
+  <h2>2. Add your child's details:</h2>
+  <form action="" method="POST">
+    <div class="form-group">
+      <label for="firstName">Child’s first name</label>
+      <input type="text" class="form-control" id="firstName" name="firstName" pattern=".*\S+.*"
+        title="First name must contain at least one letter" required />
```
Comment (mmmmillar) [inline templates/createDropbox.hbs]: Can we change this to "first name can not be empty"

File: templates/createDropbox.hbs [hunk 6]
```diff
@@ -59,73 +59,37 @@
...
+    </div>
+    <div class="form-group">
+      <label for="lastName">Child’s last name</label>
+      <input type="text" class="form-control" id="lastName" name="lastName" pattern=".*\S+.*"
+        title="Last name must contain at least one letter" required />
```
Comment (mmmmillar) [inline templates/createDropbox.hbs]: Ditto

File: lib/use-cases/SaveDropbox.js [hunk 7]
```diff
@@ -20,32 +8,19 @@ module.exports = ({ gateways: { dropbox: dropboxes } }) => {
...
-      customerDob,
-      customerEmail,
-      customerPhone,
-      customerReference,
-      customerNationalInsurance,
-      archived
-    } = fields;
+    const { description, firstName, dob, lastName, parentsEmail } = fields;
 
-    dropbox.description = validValueOrNull(description);
```
Comment (mmmmillar) [inline lib/use-cases/SaveDropbox.js:34]: We should probably keep this one in there as description is not a required field (so should be null in the db)

**Review by mmmmillar — APPROVED** [review]
Looks good, lets squash it

**Pull request merged**